src/pages/docs/platform/architecture/performance.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Performance
+meta_description: "Understand how Ably delivers messages quickly and efficiently, even at high volumes."
+---
+
+For a system such as Ably, performance generally relates to the speed of operations; for Ably the single most important metric is the latency of message delivery, but performance relates more broadly to latency and throughput of operations generally.
+
+## Why performance matters
+
+For the majority of use-cases, applications employ Ably because the timeliness of message delivery is very important. Performance in a realtime messaging platform directly affects user experience and, ultimately, business outcomes.
+
+Applications such as live chat, collaborative editing or multiplayer games depend on low latency for a responsive user experience. When messages are delayed, chat applications feel sluggish rather than instantaneous, collaborative editing tools develop conflicts that frustrate users, and multiplayer games become unplayable due to synchronization issues. Even small increases in latency can have major impacts on how users perceive application quality.
+
+From a business perspective, performance directly impacts outcomes. Research consistently shows that user engagement drops significantly with each additional 100ms of latency, conversion rates decline when applications feel sluggish, and operational costs increase when inefficient systems require more resources to achieve the same results.
+
+Low and predictable latencies are needed to underpin predictability of any system built using Ably. Designing for performance delivers an optimum experience for end-users but indirectly, by ensuring efficiency in the design, it also helps to deliver scalability. When each operation is individually performant, the system can handle more operations with the same resources.
+
+## Key performance design goals and metrics
```

Comment:
When run locally I see an extra line break under this heading. but can't see why?

Reply:
Yes, I have a bug open for this. When a heading spans the width of the page, it pushes the anchor link down a line but the padding is still added to it.